config/main.py

```diff
@@ -1515,6 +1515,18 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
             click.echo("Input {} config file(s) separated by comma for multiple files ".format(num_cfg_file))
             return
 
+    #Validate config_db.json file
```

Comment:
>Validate [](http://example.com/codeflow?start=5&length=8)

Add one blank after `#`. I notice some old code does not follow, but let's align the new code with the code style.